custom_components/kia_uvo/binary_sensor.py

```diff
@@ -37,17 +38,19 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         ("backWindowHeater", "Back Window Heater", "vehicleStatus.sideBackWindowHeat", "mdi:car-defrost-rear", "mdi:car-defrost-rear", None),
         ("steeringWheelHeater", "Steering Wheel Heater", "vehicleStatus.steerWheelHeat", "mdi:steering", "mdi:steering", None),
     ]
+    
+
 
     if vehicle.engine_type is VEHICLE_ENGINE_TYPE.EV or vehicle.engine_type is VEHICLE_ENGINE_TYPE.PHEV:
         BINARY_INSTRUMENTS.append(("charging", "Charging", "vehicleStatus.evStatus.batteryCharge", None, None, DEVICE_CLASS_BATTERY_CHARGING))
         BINARY_INSTRUMENTS.append(("pluggedIn", "Plugged In", "vehicleStatus.evStatus.batteryPlugin", None, None, DEVICE_CLASS_PLUG))
     if vehicle.engine_type is VEHICLE_ENGINE_TYPE.PHEV or vehicle.engine_type is VEHICLE_ENGINE_TYPE.IC:
         BINARY_INSTRUMENTS.append(("lowFuelLight", "Low Fuel Light", "vehicleStatus.lowFuelLight", "mdi:gas-station-off", "mdi:gas-station", None))
 
-    binary_sensors = [
-        InstrumentSensor(hass, config_entry, vehicle, id, description, key, on_icon, off_icon, device_class)
-        for id, description, key, on_icon, off_icon, device_class in BINARY_INSTRUMENTS
-    ]
+    binary_sensors = []
```

Comment:
what about like this (please pay attention to spaces/indentation)
```
for id, description, key, on_icon, off_icon, device_class in BINARY_INSTRUMENTS:
    if vehicle.get_child_value(key) != None:
        binary_sensors.append(InstrumentSensor(hass, config_entry, vehicle, id, description, key, on_icon, off_icon, device_class))
```

```

Reply:
Done.  Testing now, I also did the same for "sensors" as items like range for fuel don't exist on mine.